### PR TITLE
Changelog 20210805

### PR DESCRIPTION
**Pull request by vEpiphyte**

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -5,14 +5,100 @@ Synapse Changelog
 *****************
 
 
+v2.53.0 - 2021-08-05
+====================
+
+This release contains an automatic data migration that may cause additional
+startup time on the first boot. This is done to unique array properties which
+should previously were not uniqued. Deployments with startup or liveliness probes
```
Comment (mikemoritz) [inline CHANGELOG.rst:13]: "which should previously" -> "which previously"

File: CHANGELOG.rst [hunk 2]
```diff
@@ -5,14 +5,100 @@ Synapse Changelog
...
+should have those disabled while this upgrade is performed to prevent accidental
+termination of the Cortex process. Please ensure you have a tested backup available
+before applying this update.
+
+Features and Enhancements
+-------------------------
+- Add an ``embeds`` option to storm to allow extracting additional data about
```
Comment (mikemoritz) [inline CHANGELOG.rst:20]: either remove "about" or "about <foo>"

File: CHANGELOG.rst [hunk 3]
```diff
@@ -5,14 +5,100 @@ Synapse Changelog
...
+  when performing queries.
+  (`#2314 <https://github.com/vertexproject/synapse/pull/2314>`_)
+- Enforce node data permissions at the Layer boundary. Remove the,
```
Comment (mikemoritz) [inline CHANGELOG.rst:23]: no comma after "the"

File: CHANGELOG.rst (hunk 2 above)
Comment (mikemoritz) [reply CHANGELOG.rst:20]: And I think in general we capitalize Storm

File: CHANGELOG.rst [hunk 4]
```diff
@@ -5,14 +5,100 @@ Synapse Changelog
...
+- Warn if a LMDB commit operation takes too long.
+  (`#2316 <https://github.com/vertexproject/synapse/pull/2316>`_)
+- Add new data types, ``taxon`` and ``taxonomy``, to describe hierarchical
+  taxonomies.
+  (`#2312 <https://github.com/vertexproject/synapse/pull/2312>`_)
+- Add a new Business Development model. This allows tracking items related to
+  contract, sales, and purchasing lifecycles. This adds the following new forms
+  to the data model: ``biz:dealtype``, ``biz:prodtype``, ``biz:dealstatus``,
+  ``biz:rfp``, ``biz:deal``, ``biz:bundle``, ``biz:product``, and
+  ``biz:stake``. The Org model is also updated to futher extended to add new
```
Comment (mikemoritz) [inline CHANGELOG.rst:42]: "to further extended to add new"

File: CHANGELOG.rst [hunk 5]
```diff
@@ -5,14 +5,100 @@ Synapse Changelog
...
+  forms for supporting parts of the business lifecycle, adding ``ou:jobtype``,
+  ``ou:jobtitle``, ``ou:employment``, ``ou:opening``, ``ou:vitals``,
+  ``ou:camptype``, and ``ou:orgtype``, ``ou:conttype`` forms. The Person model
+  got a new form, ``ps:workhist``.
+  (`#2312 <https://github.com/vertexproject/synapse/pull/2312>`_)
+- Add a ``:deleted`` property to ``inet:web:post``.
+  (`#2312 <https://github.com/vertexproject/synapse/pull/2312>`_)
+- Update the following array properties to be unique sets, and add a a data
```
Comment (mikemoritz) [inline CHANGELOG.rst:50]: Two "a"s

**Review by invisig0th — APPROVED** [review]

**Pull request merged**